### PR TITLE
Day twenty eight

### PR DESCRIPTION
**Pull request by Qairkhan**
edit test for profilereducer

### CODE REVIEW

File: src/redux/actionCreators.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ const addPostCreator = (newPostText) => ({
   newPostText,
 });
 
+const deletePost = (postId) => ({
+  type: ACTION_TYPES.DELETE_POST,
+  postId,
```
Comment (Dmitry-White) [inline src/redux/actionCreators.js]: Consider keeping the action object schema as { type, payload }

File: src/redux/thunks.js [hunk 2]
```diff
@@ -3,23 +3,24 @@ import { usersAPI } from "../api/api";
...
-    usersAPI.authLogout().then((response) => {
-      if (response.data.resultCode === 0) {
-        dispatch(setUserData(null, null, null, null));
-      }
-    });
-  };
+  if (response.data.resultCode === 0) {
+    dispatch(getAuthUserData());
+  } else {
+    let mess =
```
Comment (Dmitry-White) [inline src/redux/thunks.js:20]: Don't forget about "const" where applicable

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/redux/actionCreators.js (hunk 1 above)
Comment (Qairkhan) [reply src/redux/actionCreators.js]: done

File: src/redux/thunks.js (hunk 2 above)
Comment (Qairkhan) [reply src/redux/thunks.js:20]: done

File: src/redux/actionCreators.js [hunk 3]
```diff
@@ -87,12 +92,12 @@ const getUnfollowThunkCreator = (u) => {
   };
 };
 
-const getAuthUserData = () => (dispatch) => {
-  usersAPI.authMe().then((response) => {
-    if (response.data.resultCode === 0) {
-      dispatch(setUserData(response.data.data));
-    }
-  });
+const getAuthUserData = () => async (dispatch) => {
```
Comment (Dmitry-White) [inline src/redux/actionCreators.js]: Consider moving these thunks to the thunks.js.

File: src/redux/profileReducer.js [hunk 4]
```diff
@@ -27,6 +27,13 @@ const profileReducer = (state = initialState, action) => {
       };
     }
 
+    case ACTION_TYPES.DELETE_POST: {
+      return {
+        ...state,
+        postsData: state.postsData.filter(p => p.id != action.postId),
```
Comment (Dmitry-White) [inline src/redux/profileReducer.js]: I suppose there's an error.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/redux/actionCreators.js (hunk 3 above)
Comment (Qairkhan) [reply src/redux/actionCreators.js]: done

File: src/redux/profileReducer.js (hunk 4 above)
Comment (Qairkhan) [reply src/redux/profileReducer.js]: Этот функционал не реализован, создан для удовлетворения теста

**Review by Dmitry-White — APPROVED** [review]

**Pull request merged**